<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormAbout</class>
 <widget class="QWidget" name="FormAbout">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>877</width>
    <height>597</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(250, 253, 255);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>30</number>
   </property>
   <property name="topMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>30</number>
   </property>
   <property name="bottomMargin">
    <number>30</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>首页 &gt; 关于我们</string>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnGoBack">
       <property name="minimumSize">
        <size>
         <width>82</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-color:#FAFDFF;
border: 1px solid #DFE3E5;
border-radius: 4px;
color: #2D3440;
font-size: 14px;</string>
       </property>
       <property name="text">
        <string>返回</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item alignment="Qt::AlignHCenter|Qt::AlignTop">
    <widget class="QLabel" name="label_2">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>24</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>24</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 20px;
font-weight: 500;
color: #3D3D3D;</string>
     </property>
     <property name="text">
      <string>欢迎进入厂家模式</string>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignHCenter|Qt::AlignTop">
    <widget class="QLabel" name="label_3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 14px;
font-weight: 350;
color: rgb(94, 98, 110);
background-image: url(:/image/images/warnicon.png);
background-position: left center;
background-repeat:no-repeat;
padding-left:30px;
</string>
     </property>
     <property name="text">
      <string>厂家模式仅供越疆研发内部使用，一键开启所有厂家功能</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0" columnstretch="0,0" rowminimumheight="0,0,0">
       <item row="0" column="1">
        <widget class="QLabel" name="labelRandomCode">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>44</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>44</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font-size: 16px;
font-weight: 350;
color: #2D3440;
padding-left:10px;</string>
         </property>
         <property name="text">
          <string>8130</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" alignment="Qt::AlignRight">
        <widget class="QLabel" name="labelCode">
         <property name="styleSheet">
          <string notr="true">font-size: 16px;
font-weight: 350;
color: #2D3440;</string>
         </property>
         <property name="text">
          <string>随机码：</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEditPwd">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>280</width>
           <height>44</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>280</width>
           <height>44</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border-radius: 4px;
opacity: 1;
background: #FFFFFF;
border: 1px solid #0047BB;
padding-left:10px;</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" alignment="Qt::AlignRight">
        <widget class="QLabel" name="labelPwd">
         <property name="styleSheet">
          <string notr="true">font-size: 16px;
font-weight: 350;
color: #2D3440;</string>
         </property>
         <property name="text">
          <string>密码：</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QPushButton" name="btnOk">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>44</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>44</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border-radius: 3px;
opacity: 1;
background: #0047BB;
color: #FAFDFF;
font-size: 16px;
font-weight: 350;</string>
         </property>
         <property name="text">
          <string>确定</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="verticalSpacing">
        <number>16</number>
       </property>
       <item row="0" column="0" alignment="Qt::AlignLeft">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>维护工具版本：</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" alignment="Qt::AlignRight">
        <widget class="QLabel" name="labelAppVersion">
         <property name="text">
          <string>V1.223.22</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" alignment="Qt::AlignLeft">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>工具服务版本：</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" alignment="Qt::AlignRight">
        <widget class="QLabel" name="labelToolVersion">
         <property name="text">
          <string>2.0.0sv2024d0726h0934</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" alignment="Qt::AlignLeft">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>云端配置版本：</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" alignment="Qt::AlignRight">
        <widget class="QLabel" name="labelCfgVersion">
         <property name="text">
          <string>V1.3.0</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" alignment="Qt::AlignLeft">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>使用手册版本：</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1" alignment="Qt::AlignRight">
        <widget class="QLabel" name="labelManualVersion">
         <property name="text">
          <string>V1.3.0</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" alignment="Qt::AlignLeft">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>工具日志：</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1" alignment="Qt::AlignRight">
        <widget class="QPushButton" name="btnLogExport">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>22</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>22</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">border:none;
opacity: 1;
font-size: 16px;
font-weight: 350;
line-height: 22px;
color: #0047BB;</string>
         </property>
         <property name="text">
          <string>导出日志</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="2" alignment="Qt::AlignRight">
        <widget class="QPushButton" name="btnLicense">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>22</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>22</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>-1</pointsize>
           <weight>43</weight>
           <bold>false</bold>
           <underline>false</underline>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">border:none;
opacity: 1;
font-size: 16px;
font-weight: 350;
line-height: 16.38px;
color: #0047BB;
</string>
         </property>
         <property name="text">
          <string>查看隐私协议</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
